### ISSUE TITLE
bugfix/feature request: parse more valid trailers

### ISSUE THREAD

**Issue by SKalt** [gbprod/tree-sitter-gitcommit#47]
> There can be whitespaces before, inside or after the value.
> The value may be split over multiple lines with each subsequent line starting with at least one whitespace, like the "folding" in RFC 822.
>
> -- https://git-scm.com/docs/git-interpret-trailers#_description

I think this means
```txt
subject

trailer-without-space:value
multiline-trailer: a
 b
 c
```
Should parse like
```scm
(source
  (subject)
  (message
    (trailer (token) (value))   ; trailer-without-space
    (trailer (token) (value)))) ; multiline-trailer
```
I don't have an urgent to parse multiline / oddly-shaped trailers, and I'm not sure I'll have time to submit this in the next month or so. Feel free to close this issue as not needed or tag it as a nice-to-have.

**Comment by gbprod** [comment]
Hey,
That's really interesting! I can't work there right now because I'm on vacation without any computer 😅
I keep that in mind for when I come back

**Comment by gbprod** [comment]
Fixed by #49
